.ci/docker/ubuntu-bionic

```diff
@@ -0,0 +1,18 @@
+FROM ubuntu:bionic
```

Comment:
Let's either use the prefix or suffix `Dockerfile` here to indicate this filetype.
`Dockerfile.ubuntu-bionic` is one common convention.